source/code/plugin/health/health_container_cpu_memory_aggregator.rb

```diff
@@ -163,11 +175,24 @@ def get_records
             container_cpu_memory_records = []
 
             @cpu_records.each{|resource_key, record|
+
+                cpu_limit_mc = 1.0
+                if record["limit"].is_a?(Numeric)
+                    cpu_limit_mc = record["limit"]/1000000.to_f
+                else
+                    @log.info "CPU Limit is not a number #{record['limit']}"
+                    if !@@limit_is_array_event_sent
```

Comment:
Is this an aggregation to send this event only once?
If so, we will send this event either for container/node for cpu/memory.. is that what is intended?

Reply:
I slightly changed the logic to send it once per workload per cluster.